solutions/combining-data-fetching-strategies/pages/index.tsx

```diff
@@ -74,40 +75,49 @@ function ProductCard({product}: Props) {
           <div className="flex justify-between w-full items-baseline">
             <div className="ml-4 mr-auto text-left flex flex-col">
               <h4 className="font-semibold text-xl">{product.title}</h4>
-              <h5 className="text-gray-700">{product.description} ({stock} left)</h5>
+              <h5 className="text-gray-700">{product.description} {isLoading ? `` : `(${stock} left)`}</h5>
             </div>
             <h4 className="font-bold text-lg">USD {product.price}</h4>
           </div>
         </div>
         <div className="p-4 gap-4 flex flex-col justify-center items-center border-b">
-          {stock > 0 ? isAdded ? (
+          {isLoading ? (
             <a
               role="button"
-              rel="noopener noreferrer"
-              target="_blank"
-              className="py-4 px-6 text-lg w-full bg-green-500 text-center text-white rounded-md"
-            >
-              Added!
-            </a>
-          ) : (
-            <a
-              role="button"
-              onClick={() => toggleAdded(true)}
-              rel="noopener noreferrer"
-              target="_blank"
-              className="py-4 px-6 text-lg w-full bg-black text-center text-white hover:text-white rounded-md hover:bg-gray-900"
-            >
-              Add to cart
-            </a>
-          ) : (
-            <a
-              role="button"
-              rel="noopener noreferrer"
-              target="_blank"
               className="py-4 px-6 text-lg w-full bg-gray-500 cursor-not-allowed disabled text-center text-white rounded-md"
             >
-              No stock available
+              Loading...
             </a>
+          ) : (
+            <>
+              {isAdded ? (
+                <a
+                  role="button"
+                  className="py-4 px-6 text-lg w-full bg-green-500 text-center text-white rounded-md"
+                >
+                  Added!
+                </a>
+              ) : (
+                <>
+                  {stock > 0 ? (
+                    <a
+                      role="button"
+                      onClick={() => toggleAdded(true)}
+                      className="py-4 px-6 text-lg w-full bg-black text-center text-white hover:text-white rounded-md hover:bg-gray-900"
+                    >
+                      Add to cart
+                    </a>
+                  ) : (
+                    <a
+                      role="button"
+                      className="py-4 px-6 text-lg w-full bg-gray-500 cursor-not-allowed disabled text-center text-white rounded-md"
+                    >
+                      No stock available
```

Comment:
to avoid comparing to a stock like what we have at the stock exchange we could use `No stock photo available`?

Reply:
This is about stock units